service/auth/auth.go

```diff
@@ -341,25 +347,33 @@ func hashPassword(password string) (string, error) {
 }
 
 // CreateDefaultUser creates a default user in the database
-func (Service) CreateDefaultUser(username string, password string) {
-	db := persistence.GetDatabase()
-
-	var count int64
-	db.Model(&auth.User{}).Count(&count)
+// TODO(all): return created user?
+func (s *Service) CreateDefaultUser(username string, password string) error {
+	var (
+		storedUsers []*auth.User
+		err         error
+	)
 
-	if count == 0 {
+	err = s.db.Read(&storedUsers)
```

Comment:
Can we have a `Count()` function in the database interface please :) This seems very complicated and it is not necessary to fetch all users.